### PR TITLE
Run "npm install" to fix missing js-yaml issue

### LINKED ISSUE #585: Missing dependency js-yaml
The lastest docker image (develop) seems to be missing a dependecy, even though it is listed in packages.json:
```
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'js-yaml'
Require stack:
- /usr/src/app/node_modules/matrix-appservice/lib/app-service-registration.js
- /usr/src/app/node_modules/matrix-appservice/lib/index.js
- /usr/src/app/node_modules/matrix-appservice-bridge/lib/components/cli.js
- /usr/src/app/node_modules/matrix-appservice-bridge/lib/index.js
- /usr/src/app/lib/app.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/usr/src/app/node_modules/matrix-appservice/lib/app-service-registration.js:8:35)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/src/app/node_modules/matrix-appservice/lib/app-service-registration.js',
    '/usr/src/app/node_modules/matrix-appservice/lib/index.js',
    '/usr/src/app/node_modules/matrix-appservice-bridge/lib/components/cli.js',
    '/usr/src/app/node_modules/matrix-appservice-bridge/lib/index.js',
    '/usr/src/app/lib/app.js'
  ]
}
```

### PR DESCRIPTION
**Pull request by jaller94**
Fixes #585

@erdnaxeli Are you able to confirm that this fixes the issue?

### CODE REVIEW

File: changelog.d/586.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Run "npm install" to fix missing js-yaml issue
```
Comment (Half-Shot) [inline changelog.d/586.misc:1]: Probably would prefer something like `Regenerate package-lock.json to fix a build issue`

**Review by Half-Shot — APPROVED** [review]
Gah, that's really annoying. Thanks

File: changelog.d/586.misc (hunk 1 above)
Comment (jaller94) [reply changelog.d/586.misc:1]:
```suggestion
Regenerate package-lock.json to fix a build issue
```

**Comment by erdnaxeli** [comment]
Sorry I didn't had time to test the fix :sweat_smile: 

All is good now with the latest docker image, thanks!

**Pull request merged**